conbench/api/_docs.py

```diff
@@ -78,7 +78,6 @@ def _201_created(example, schema=None):
 spec.components.response("CommitList", _200_ok([ex.COMMIT_ENTITY]))
 spec.components.response("CompareEntity", _200_ok(ex.COMPARE_ENTITY))
 spec.components.response("CompareList", _200_ok(ex.COMPARE_LIST))
-spec.components.response("CompareBenchmarkResult", _200_ok(ex.COMPARE_BENCHMARK_RESULT))
```

Comment:
This was unused.

Reply:
We do still have the ability to compare benchmark results today, but that entity is represented by `CompareEntity` in docs, not `CompareBenchmarkResult`. The latter was a legacy object from a while ago!

Reply:
>https://conbench.ursa.dev/api/compare/benchmarks/805383b072ac488db96ef1ec6bd941c3...8803a118ab6245b7b244d6227ffd9530/ returns a 404 for me

We also renamed the endpoint - https://conbench.ursa.dev/api/compare/benchmark-results/805383b072ac488db96ef1ec6bd941c3...8803a118ab6245b7b244d6227ffd9530/ works.

Reply:
Awesome, thanks for that! And sorry I forgot about the renaming